ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/networking/libp2p/rpc/BlobSidecarsByRootRequestMessageSchema.java

```diff
@@ -20,8 +20,8 @@
 public class BlobSidecarsByRootRequestMessageSchema
     extends AbstractSszListSchema<BlobIdentifier, BlobSidecarsByRootRequestMessage> {
 
-  // Size validation according to the spec (MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK) is
-  // done in the RPC handler
+  // Size validation according to the spec (MAX_REQUEST_BLOCKS_DENEB *
```

Comment:
This message means we don't throw in constructor if it exceeds (MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK), we will have spec check later. So here we cap it by 2048 and later check that it's 6, so this comment is made for understanding why don't we cap here on 6 and where is actual validation.
So I'd revert this file change.

Reply:
Oh, I see. I misread that. Will revert that change!

Reply:
this line where actual validation happens
https://github.com/Consensys/teku/blob/94d37825e1fb4e2f14c3e964fa8eb68d5f8e4f1b/networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRootMessageHandler.java#L86